www/js/services.js

```diff
@@ -15,15 +15,35 @@ angular.module('emission.services', ['emission.plugin.logger',
             window.cordova.plugins.BEMServerComm.getUserPersonalData("/profile/create", successCallback, errorCallback);
     };
 
+    var processErrorMessages = function(errorMsg) {
+      var titleMsg = "Error"
+      if (errorMsg.includes("403")) {
+        titleMsg = "Error: OPcode not found";
+        errorMsg = "Error: OPcode not found. " + errorMsg;
+        console.error("Error 403 found. " + errorMsg);
+      }
+      return errorMsg;
+    }
+
     this.updateUser = function(updateDoc) {
         return new Promise(function(resolve, reject) {
             window.cordova.plugins.BEMServerComm.postUserPersonalData("/profile/update", "update_doc", updateDoc, resolve, reject);
+        })
+        .catch(error => {
+          error = "While updating user, " + error;
```

Comment:
Do you need the "While updating user" etc comments? The popup already has a description of when we got the error, right?

Reply:
To keep it standard with all the other function error catching blocks, I think this might be helpful.

More error message info is better than less.

I will test one case of the error 403 real quick (not necessarily the `updatingUser` function) to see how it displays with the error message information and then make a decision with whether it's helpful or not